prac_05/emails.py

```diff
@@ -0,0 +1,21 @@
+"""
+Emails
+CP1404/CP5632 Practical
+Estimate: 30 minutes
+Actual: 45 minutes
+A program that takes name from users email and add to dictionary.
+"""
+
+name_to_email = {}
+email = input("email: ")
+while email != "":
+    names = email[:email.index("@")].split(".")  # Creating a list of first and last names.
```

Comment:
Could've used a function for extracting name from email, although not super necessary in this case